src/Raven.Studio/wwwroot/Content/css/icons.less

```diff
@@ -459,7 +459,7 @@
 
 /* MENU AND TOPBAR ICON COLORS*/
 
-#main-menu, .inputs-container, .stats {
+#main-menu, .inputs-container {
```

Comment:
why?

Reply:
This fixes the problem that the (i) was purple
With this fix, the icon is back to blue, and it is only purple in the main 'About'

**See comment in**
https://issues.hibernatingrhinos.com/issue/RavenDB-16067#focus=Comments-67-54400.0-0


![image](https://user-images.githubusercontent.com/17376974/103533066-1edfa900-4e95-11eb-85c4-7a1208eec1fa.png)

Reply:
So I reverted this change and will update the comment in issue RavenDB-16067